src/Compute/Compute/help/Add-AzVmssSshPublicKey.md

```diff
@@ -25,8 +25,8 @@ The **Add-AzVmssSshPublicKey** cmdlet adds the public keys that you can use to c
 
 ### Example 1: Add an SSH public key to the VMSS
 ```
```

Comment:
Please add `powershell` at the begin of the block

Reply:
Done. Please check.